RELEASING.md.template

```diff
@@ -1,27 +1,43 @@
 # Releasing
 
-1. Update version file accordingly.
-1. Update `NEWS.md` or `CHANGELOG.md` to reflect the changes since last release.
-1. Commit changes.
-   There shouldn't be code changes,
-   and thus CI doesn't need to run,
-   you can then add "[ci skip]" to the commit message.
-1. Tag the release: `git tag -s vVERSION`
-    - We recommend the [_quick guide on how to sign a release_] from git ready.
-1. Push changes: `git push --tags`
-1. Build and publish:
-    ```bash
-    gem build project-name.gemspec
-    gem push project-name-*.gem
-    ```
-1. Add a new GitHub release using the recent `NEWS.md` or `CHANGELOG.md` as the content. Sample
-   URL: https://github.com/thoughtbot/project-name/releases/new?tag=vVERSION
-1. Announce the new release,
-   making sure to say "thank you" to the contributors
-   who helped shape this version!
+## 1. Update version file accordingly
+
+Include an RC (release candidate) number if appropriate, e.g. 2.0.0.rc.
+
+## 2. Update `NEWS.md` or `CHANGELOG.md` to reflect the changes since last release
+
+You can copy [GitHub automatically generated release notes] for this step.
+
+## 3. Commit changes
+
+There shouldn't be code changes, and thus CI doesn't need to run. You can then add "[ci skip]" to the commit message.
```

Comment:
Shortening this a little to make it clear that the reader should do this (and not have to think about if they should because of the "you can"):

```
There shouldn't be code changes, and thus CI doesn't need to run. Add "[ci skip]" to the commit message.
```

Reply:
👍🏼